src/Tests/Microsoft.NET.Sdk.BlazorWebAssembly.Tests/Microsoft.NET.Sdk.BlazorWebAssembly.Tests.csproj

```diff
@@ -1,24 +1,22 @@
-<Project Sdk="Microsoft.NET.Sdk">
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the MIT license. See License.txt in the project root for full license information. -->
```

Comment:
why this change is needed?

Reply:
I was following the convention in some of [the](https://github.com/dotnet/sdk/blob/83d9d6971af6cdf68e409f7f9baa5501093f5018/src/Tests/Microsoft.NET.Restore.Tests/Microsoft.NET.Restore.Tests.csproj) [other](https://github.com/dotnet/sdk/blob/83d9d6971af6cdf68e409f7f9baa5501093f5018/src/Tests/Microsoft.NET.Build.Tests/Microsoft.NET.Build.Tests.csproj) [test](https://github.com/dotnet/sdk/blob/83d9d6971af6cdf68e409f7f9baa5501093f5018/src/Tests/Microsoft.NET.Pack.Tests/Microsoft.NET.Pack.Tests.csproj) [project](https://github.com/dotnet/sdk/blob/83d9d6971af6cdf68e409f7f9baa5501093f5018/src/Tests/Microsoft.NET.ToolPack.Tests/Microsoft.NET.ToolPack.Tests.csproj) [files](https://github.com/dotnet/sdk/blob/83d9d6971af6cdf68e409f7f9baa5501093f5018/src/Tests/Microsoft.NET.Clean.Tests/Microsoft.NET.Clean.Tests.csproj). I noticed that some projects use `<Project Sdk="">` directly. I believe that import enforces importing the props file first and the targets file last so it shouldn't be functionally different from what is here.